go/tendermint/apps/roothash/state/state.go

```diff
@@ -83,6 +89,17 @@ func (s *ImmutableState) Runtimes() []*RuntimeState {
 	return runtimes
 }
 
+func (s *ImmutableState) RoundTimeout() (time.Duration, error) {
+	_, raw := s.Snapshot.Get(roundTimeoutKeyFmt.Encode())
+	if raw == nil {
```

Comment:
Hmmm, should this just bail if it is missing? (likewise for the other methods...)

We really need sanity checking of the genesis state at some point. :(

Reply:
Yeah the improved sanity checks and genesis tooling will be done next in #1633.

Reply:
But otherwise yeah, I think these should still panic instead of returning a default (and the default should be set in `InitChain` instead).